nixos/modules/services/networking/ntpd.nix

```
@@ -98,14 +116,21 @@ in
   config = mkIf config.services.ntp.enable {
 
     # Make tools such as ntpq available in the system path.
-    environment.systemPackages = [ pkgs.ntp ];
+    environment.systemPackages = [ config.services.ntp.package ];
```

In line 9 it's aliased to `cfg` like (pretty much) all modules do it. Then it's immediately obvious that the config comes from the current package and not another one.

```suggestion
    environment.systemPackages = [ cfg.package ];
```

---

Good idea!